scripts/west_commands/sign.py

```diff
@@ -220,7 +220,7 @@ def sign(self, command, build_dir, build_conf, formats):
         flash = self.edt_flash_node(b, args.quiet)
         align, addr, size = self.edt_flash_params(flash)
 
-        if build_conf.getboolean('CONFIG_BOOTLOADER_MCUBOOT'):
+        if build_conf.getboolean('CONFIG_BOOTLOADER_MCUBOOT') == False:
```

Comment:
```suggestion
        if not build_conf.getboolean('CONFIG_BOOTLOADER_MCUBOOT'):
```

Just a style nit, but feel free to ignore this if you'd prefer to leave as is.